psi4/src/psi4/libmints/pointgrp.h

```diff
@@ -157,13 +160,17 @@ class SymmetryOperation
     double trace() const
     { return d[0][0] + d[1][1] + d[2][2]; }
 
+    // For the sake of Python export, expose d to reading
+    const std::array<std::array<double, 3>, 3>& matrix()
+    { return d; }
```

Comment:
I see these are historical, but this should be on a single line. `const std::array<std::array<double, 3>, 3>& matrix() { return d; }`. Can either fix now or when c-f gets it next pass.

Reply:
I'll leave this as part of next week's great clang-formatting. Breaking consistency would make the section less readable IMO, and formatting pointgrp would clutter this PR.